packages/querygrid/src/components/user/actions.ts

```diff
@@ -0,0 +1,107 @@
+import moment from 'moment'
+import { Map, OrderedMap } from 'immutable'
+import { Ajax , ActionURL, Utils } from '@labkey/api'
+import { caseInsensitive, hasAllPermissions, PermissionTypes, SchemaQuery, User, buildURL } from "@glass/base";
+import { ChangePasswordModel } from "./models";
+
+export function getUserPermissionsDisplay(user: User): Array<string> {
+    let permissions = [];
+
+    if (user.isAdmin) {
+        permissions.push('Administrator');
```

Comment:
Eventually we'll want this to be "Power User", I believe.

Reply:
Yes, I was going to wait for the SM user permissions story to touch that part.